README.md

```diff
@@ -3,6 +3,15 @@
 This repository contains the `mirsg.infrastructure` Ansible Collection. This collection can be used to
 configure infrastructure for deploying XNAT and OMERO.
 
+## Included content
+
+### Playbooks
+
+| Name                                                         | Description                                                |
+| ------------------------------------------------------------ | ---------------------------------------------------------- |
+| [setup_user_accounts.yml](playbooks/setup_user_accounts.yml) | Create OS user accounts a group of servers `target`, which |
+| defaults to `all`.                                           |
```

Comment:
Is this missing a final `|` or is markdown magic?

Reply:
ah I tried to split to second column over multiple lines as it was a really long line, but then prettier added some extra `|` to make it two rows. I've put it all in one line now

Reply:
IMO line wrapping is okay for tables - and I'm big fan of prettier's table work

Reply:
yeah prettier's table formatting is really nice (so long as you have a big screen!)

Reply:
To be fair, it's only a problem when not rendered 😛